run_dir/static/css/status.css

```diff
@@ -26,35 +26,19 @@ multiple places and typically control typography, or other common elements.
 
 
 /* index.html page */
-#project_quotas .plot-column {
-	height: 60px;
-	padding: 0;
-	overflow: hidden;
-}
-
-#project_quotas tr td:first-child {
-	width: 150px;
-}
-
-.update_table {
-	overflow: hidden;
-}
-
-#project_quotas td {
+.table-disk-quotas > tbody > tr > td {
   padding: 0;
 }
-#project_quotas td:first-child {
```

Comment:
You can also add the `width: 150px;` that was in the statement higher up, which prevents the column from getting weirdly narrow.